long-ride-alerts/src/main/java/org/apache/flink/training/exercises/longrides/LongRidesExercise.java

```diff
@@ -98,17 +99,66 @@ public static void main(String[] args) throws Exception {
     @VisibleForTesting
     public static class AlertFunction extends KeyedProcessFunction<Long, TaxiRide, Long> {
 
+        private static final long serialVersionUID = -7341825009875511205L;
+        public static final long TWO_HOURS = 120 * 60 * 1000;
```

Comment:
Would be better to use something like this
`Duration.ofHours(2).toMillis();`
or
`Time.hours(2).toMilliseconds();`

Reply:
Sure, thanks!!